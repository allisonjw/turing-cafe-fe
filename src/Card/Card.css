.card {
    background-color: white;
    border: 5px solid teal;
    height: max-content;
    width: 15%;
    margin: 1%;
    padding: 1%;
}

.card_name, 
.card_date,
.card_time,
.card_number,
p {
    color: teal;
}

.card_btn {
    background-color: teal;
    height: 26px;
    width: 93%;
    color: white;

}