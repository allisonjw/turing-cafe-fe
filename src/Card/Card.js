import React from 'react';
import './Card.css';

const Card = ({id, name, date, time, number}) => {
    return (
        <div className="card">
          <h2 className="card_name">{name}</h2>
          <p className="card_date">{date}</p>
          <p className="card_time">{time}</p>
          <p className="number">{number}</p>
        </div>
    )
}

export default Card;