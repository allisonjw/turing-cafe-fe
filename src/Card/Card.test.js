import React from 'react';
import {shallow} from 'enzyme';
import Card from './Card';

describe('Card', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Card />)
  });

  it('should match all the data being passed', () => {
    expect(wrapper).toMatchSnapshot()
  });
  
});