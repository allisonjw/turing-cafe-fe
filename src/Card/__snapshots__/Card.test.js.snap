// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match all the data being passed 1`] = `
<div
  className="card"
>
  <h2
    className="card_name"
  />
  <p
    className="card_date"
  />
  <p
    className="card_time"
  >
     pm
  </p>
  <p
    className="number"
  >
    Number of Guest:
  </p>
  <button
    className="card_btn"
  >
    Cancel
  </button>
</div>
`;
